agents/process/process_logger.go

```diff
@@ -64,6 +67,7 @@ func (pl *processLogger) Write(p []byte) (n int, err error) {
 			return
 		}
 		line = strings.TrimSuffix(line, "\n")
+		line = pl.clearKeywords(line)
```

Comment:
AFAIK, in repeated usage, a Replacer is faster than many calls to Replace.
So maybe create Replacer outside of `for {}` will be better here.

Reply:
TIL, thanks for sharing